io-uring-test/src/main.rs

```diff
@@ -2,21 +2,47 @@
 mod utils;
 mod tests;
 
-use io_uring::{IoUring, Probe};
+use io_uring::{cqueue, squeue, IoUring, Probe};
 
 pub struct Test {
     probe: Probe,
     target: Option<String>,
 }
 
 fn main() -> anyhow::Result<()> {
-    let mut ring = IoUring::new(8)?;
+    let entries = 8;
+
+    test::<squeue::Entry, cqueue::Entry>(IoUring::new(entries)?)?;
+
+    #[cfg(feature = "unstable")]
```

Comment:
I think it also adds `not(feature = "ci")`

Reply:
Oops, fixed.